src/perf_context.rs

```diff
@@ -53,6 +59,179 @@ pub fn set_perf_level(level: PerfLevel) {
     }
 }
 
+pub struct PerfFlags {
+    inner: NonNull<DBPerfFlags>,
+}
+
+unsafe impl Send for PerfFlags {}
+
+unsafe impl Sync for PerfFlags {}
```

Comment:
Are they needed?

Reply:
I think so, otherwise it would be quite hard to use. If it's not `Send` and `Sync`, any closure that captures a `PerfFlags` or `&PerfFlags` will become `!Send`, while it's very common to require a closure to be `Send`.

Reply:
Okay.